lib/proto.js

```diff
@@ -91,13 +92,31 @@ function child (bindings) {
       instance[serializersSym][bks] = bindings.serializers[bks]
     }
   } else instance[serializersSym] = serializers
+  if (bindings.hasOwnProperty('formatters')) {
+    instance[formattersSym] = Object.create(null)
```

Comment:
there's no need, in my mind, to create a null proto object.

Reply:
I despise them.

Reply:
I'm doing the same in the default options, I thought it was your preference because you are doing it in the serializers as well.
https://github.com/delvedor/pino/blob/dafb6940f7bb6dbc31afff247987369484616f17/pino.js#L50-L60

I'll remove it :)